features/step_definitions/previous_winners_steps.rb

```diff
@@ -0,0 +1,100 @@
+When(/^I visit the previous winners page$/) do
+  visit "/auctions/winners/"
+end
+
+When(/^I visit the previous winners archive page$/) do
+  visit "/auctions/winners/archive/"
+end
+
+Then(/^I should visit the previous winners archive page$/) do
```

Comment:
I **think** this is wrong. "Should" implies an assertion to be tested. I think you might be using this to test a redirect?